JS/Lesson15/sinan/script.js

```diff
@@ -133,14 +181,107 @@ checkIfStringIsUpperCase('write a program that takes in');
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+function checkObese(weight, height){
+	const bmi = weight / (height^2);
+	if (bmi < 18.5){
+		return 'Underweight';
+	}
+	else if (bmi > 18.5 && bmi < 24.9 ){
+		return 'Healthy Weight';
+	}
+	else if (bmi > 25 && bmi < 29.9 ){
+		return 'Overweight';
+	}
+	else if (bmi > 30 ){
+		return 'Obesity';
+	}
+}
+console.log('Exercise 8:',checkObese(85,1.82));
+
+
 // 9. Write a program that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
 
+function checkAge(personAge){
+	
+	if (personAge >= 0 && personAge <= 12){
+		return 'Child';
+	}
+	else if (personAge >= 13 && personAge <= 19){
+		return 'Teenager';
+	}
+	else if (personAge >= 20 && personAge <= 64){
+		return 'Adult';
+	}
+	else if (personAge > 64){
+		return 'Senior';
+	}
+	else {
+		return 'Please insert number greater than 0';
+	}
+}
+console.log('Exercise 9:',checkAge(65));
+
+
 // 10. Write a program that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+let studentStatus = Boolean;
```

Comment:
Boolean is a type of value in JS, a declaration like this does not really make sense. Instead, you'd need to do either  `let studentStatus = true;` or `let studentStatus = false;` but I don't think you even need this variable for the exercise.